### ISSUE TITLE
Breaks Google image search and People also ask

### ISSUE THREAD

**Issue by BaldEagle58** [cowlicks/privacypossum#24]
When I have this add-on installed in Firefox 59.0.3 I can no longer click on images found by searching GOOGLE IMAGES.

A similar thing happen when I search GOOGLE and it displays a "People also ask" box, when you click on the down arrow nothing happens.

I tried the DISABLE button in Privacy Possum but that did not help, 
Everything was back to normal as soon as I removed the add-on.

**Comment by cowlicks** [comment]
@BaldEagle58 hello, what version of privacy possum was this on? You can check in the about addons page. The broken version was 2018.5.6 the fixed version is 2018.5.7.

Does 2018.5.7 fix this issue for you?

**Comment by BaldEagle58** [comment]
Hi cowlicks

And thanks for the fast response.

I can confirm that I had 2018.5.6 (with automatic updates on) but after your email I did a manual update and it downloaded 2018.5.7 which is working fine, it has resolved both the issues I mentioned.

THANKS

Sent from Mail for Windows 10

**Comment by mazesy** [comment]
Reporting latest _Privacy Possum_ version 2018.5.7 (I had commented on another thread, closed since).

DuckDuckGo : OK
Qwant : OK
Another [site](https://particulier.edf.fr) which partially broke with previous version : OK

A problem but not an issue given the page displayed correctly once I had disabled PP for that page: its the page that opens when the user wishes to log into his Firefox Account on AMO and starts with,
https://accounts.firefox.com/oauth/signin?scope=xxxxxx
Page wouldn't open correctly. I disabled PP for that site, reloaded the page and it displayed correctly. It's not an issue because PP's built-in whitelist option worked correctly.

Now we're getting to somethin' -- Keep up the good work @cowlicks
